go.mod

```diff
@@ -1,3 +1,5 @@
+replace github.com/Bios-Marcel/cordless => github.com/0xSteeW/cordless v0.0.0-20201117104521-a2cc7de4b6dd
```

Comment:
This should not be upstream, it shouldn't be necessary at all.

Reply:
Will not build either way. It was a quick test to try to get appveyor to recognize my files